src/main/java/science/icebreaker/config/ApplicationNeo4jConfigurationProperties.java

```diff
@@ -0,0 +1,58 @@
+package science.icebreaker.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+
+@Configuration
+@ConfigurationProperties(prefix = "icebreaker.neo4j")
+public class ApplicationNeo4jConfigurationProperties {
+
+    private String host;
+
+    private int port;
+
+    private String username;
+
+    private String password;
+
+
+    public String getHost() {
```

Comment:
Adding public getters *and* setters for private fields is pointless if they only directly reset the object attribute. 
However, I am not going to fight over this, if all of the code follows this convention.

Reply:
yeah.. but that's java.. I think that the Spring framework even expects getters and setters.